docs/ossm/ambient/README.md

```diff
@@ -0,0 +1,340 @@
+
+**Procedure**
+
+1.  In the OpenShift Container Platform web console, click **Operators** > **Installed Operators**.
+2.  Select `ztunnel` in the **Project** drop-down menu.
+3.  Click the Service Mesh Operator.
+4.  Click **ZTunnel**.
+5.  Click **Create ZTunnel**.
+6.  Ensure that the name is `default`.
+7.  Select the `ztunnel` project from the **Namespace** drop-down menu.
+10. Click **Create**. This action deploys the Istio control plane.  
+    When `State: Healthy` appears in the **Status** column, Istio is successfully deployed.
+
+#### 3.4.3 Creating the Istio CNI resource using the CLI
+
+TBD
+
+### 3.5 Scoping the Service Mesh with discovery selectors
+
+Service Mesh in Istio ambient mode includes workloads that meet the following criteria:
+
+* The control plane has discovered the workload.
+* The workload has been [labelled appropriately](https://istio.io/latest/docs/ambient/usage/add-workloads/), such that traffic redirection with ZTunnel has been configured.
+
+By default, the control plane discovers workloads in all namespaces across the cluster, with the following results:
+
+* Each proxy instance receives configuration for all namespaces, including workloads not enrolled in the mesh.
+* Any workload with the appropriate pod or namespace injection label receives a proxy sidecar. 
+
+In shared clusters, you might want to limit the scope of Service Mesh to only certain namespaces. This approach is especially useful if multiple service meshes run in the same cluster.
+
+#### 3.5.1 About discovery selectors
+
+With discovery selectors, the mesh administrator can control which namespaces the control plane can access. By using a Kubernetes label selector, the administrator sets the criteria for the namespaces visible to the control plane, excluding any namespaces that do not match the specified criteria.
+
+The `discoverySelectors` field accepts an array of Kubernetes selectors, which apply to labels on namespaces. You can configure each selector for different use cases:
+
+* Custom label names and values. For example, configure all namespaces with the label istio-discovery=enabled.
+* A list of namespace labels by using set-based selectors with OR logic. For instance, configure namespaces with istio-discovery=enabled OR region=us-east1.
+* Inclusion and exclusion of namespaces. For example, configure namespaces with istio-discovery=enabled AND the label app=helloworld.
+
+#### 3.5.2 Scoping a Service Mesh by using discovery selectors
+
+If you know which namespaces to include in the Service Mesh, configure `discoverySelectors` during or after installation by adding the required selectors to the `meshConfig.discoverySelectors` section of the `Istio` resource. For example, configure Istio to discover only namespaces labeled `istio-discovery=enabled`.
+
+**Prerequisites**
+
+* The OpenShift Service Mesh operator is installed.
+* An Istio CNI resource is created.
```

Comment:
Why only "Istio CNI"? 

I think you missed including the Istiod and Ztunnel here.

Reply:
Updated.